Documentation/ceph-monitoring.md

```diff
@@ -196,4 +196,27 @@ spec:
 labels:
   monitoring:
     prometheus: k8s
-[...]
\ No newline at end of file
+[...]
+```
+
+## Prometheus to auto grow OSDs
```

Comment:
I'd suggest we add this topic in ceph-advanced-configuration.md. It will just mention prometheus is a prereq and can link back to this doc if needed.
```suggestion
## Auto Expansion of OSDs
```